### PR TITLE
MEN-6083

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,12 +28,12 @@ RUN git clone https://github.com/mendersoftware/mender.git . && \
...
+ARG MENDER_APP_UPDATE_MODULE_VERSION
 COPY --from=cli-builder /go/src/github.com/mendersoftware/mender-cli /usr/bin/
 COPY --from=artifact-builder /go/src/github.com/mendersoftware/mender-artifact/mender-artifact /usr/bin/
 COPY --from=client-builder /install-modules-gen/usr/bin/ /usr/bin/
 # The --mount instead of COPY is used to decrease the image layer size. Requires: 'export DOCKER_BUILDKIT=1'
-RUN --mount=type=bind,source=deb-requirements.txt,target=/deb-requirements.txt \
-    apt-get update && \
-    apt-get install -y $(cat deb-requirements.txt) && \
-    rm -rf /var/lib/apt/lists/*
+RUN wget https://raw.githubusercontent.com/mendersoftware/app-update-module/$MENDER_APP_UPDATE_MODULE_VERSION/gen/app-gen -O /usr/bin/app-gen && chmod +x /usr/bin/app-gen
```
Comment (merlin-northern) [inline Dockerfile:37]: in all the remaining cases we are cloning repos, more or less idle comment: shall we clone or shall we wget?

**Review by merlin-northern — APPROVED** [review]
one take-it-or-leave-it comment, I am not an expert on this repo, so I would be happy if @lluiscampos could review as well.

File: Dockerfile (hunk 1 above)
Comment (tranchitella) [reply Dockerfile:37]: don't know, let's wait for @lluiscampos and see what he thinks about it

Comment (lluiscampos) [reply Dockerfile:37]: Both approaches are fine.

I favor the existing approach in that it encapsulates (part of) the complexity by using the `Makefile` as the interface. But one can also argue that it is unnecessary and that we anyway leak the paths for the executable, so doing `wget` directly on the source is not that off after all.

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ARG MENDER_CLI_VERSION=1.11.0
 ARG MENDER_ARTIFACT_VERSION=3.10.1
 ARG MENDER_CLIENT_VERSION=3.5.1
+ARG MENDER_APP_UPDATE_MODULE_VERSION=master
```
Comment (lluiscampos) [inline Dockerfile:4]: We should set to 1.0.0 once it is released.

I personally would wait for this task to be done after the tagging - but I was not part of the planning and didn't provide such input, so it is okey too to merge as-is and have a follow-up 0SP task for this.

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**